snuba/migrations/snuba_migrations/metrics/0030_metrics_distributions_v2_writing_mv.py

```diff
@@ -39,7 +39,7 @@ def forwards_local(self) -> Sequence[operations.SqlOperation]:
                 ),
                 metric_type="distribution",
                 target_mat_version=4,
-                appended_where_clause="AND timestamp > toDateTime('2022-03-29 00:00:00')",
+                appended_where_clause="AND timestamp >= toDateTime('2022-03-22 00:00:00')",
```

Comment:
Separately to the tests, can you explain what this date actually means? Like, what is different before and after this date?

Reply:
It was just a random marker so that Filippo could only backfill to that point and we know any row with a timestamp after that comes from the materialized view